### PR TITLE
updated for list check bug in predict/predict_on_batch

### PR DESCRIPTION
**Pull request by braingineer**
i think this will do it

### CODE REVIEW

File: keras/legacy/models.py [hunk 1]
```diff
@@ -473,6 +473,8 @@ def predict(self, data, batch_size=128, verbose=0):
         x = self._get_x(data)
         output_list = super(Graph, self).predict(x, batch_size=batch_size,
                                                  verbose=verbose)
+        if not isinstance(output_list,list):
```
Comment (tboquet) [inline keras/legacy/models.py]: + space for the pep8 checks :)!

**Comment by braingineer** [comment]
ref: #2551 #2580

File: keras/legacy/models.py (hunk 1 above)
Comment (braingineer) [inline keras/legacy/models.py]: yikes. thanks!

File: keras/legacy/models.py [hunk 2]
```diff
@@ -528,6 +530,8 @@ def test_on_batch(self, data, sample_weight={}, **kwargs):
 
     def predict_on_batch(self, data):
         output_list = super(Graph, self).predict_on_batch(data)
+        if not isinstance(output_list,list):
```
Comment (tboquet) [inline keras/legacy/models.py]: same here!

Comment (braingineer) [inline keras/legacy/models.py]: ya it was a copy-paste. fixed.   You meant in the function call, right?

`(output_list,list)` -> `(output_list, list)`

Comment (tboquet) [inline keras/legacy/models.py]: Yep!

Comment (braingineer) [inline keras/legacy/models.py]: ah k. Thanks!

**Comment by fchollet** [comment]
Looks good to me. Thanks!

**Pull request merged**